### PR TITLE
Add CSV Booked Days and JSON Availability Map adapters

### PR DESCRIPTION
**Pull request by ZenCocoon**

### CODE REVIEW

File: spec/vacation_rentals_ical_adapters/adapters/csv_booked_days_adapter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  describe "#applicable?" do
+    it "returns true for source with .booked_days extension regardless of the body" do
+      source_for_csv_booked_days = "http://example.com/file.booked_days"
+      expect(adapter.applicable?(body: "", source: source_for_csv_booked_days)).to eq true
+      expect(adapter.applicable?(body: "", source: "whatever")).to eq false
+    end
+  end
+
+  describe "#to_ical" do
+    it "returns iCal formated response" do
```
Comment (Azdaroth) [inline spec/vacation_rentals_ical_adapters/adapters/csv_booked_days_adapter_spec.rb]: A small nitpick, I think it should be `formatted`.

File: spec/vacation_rentals_ical_adapters/adapters/json_availability_map_adapter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  describe "#applicable?" do
+    it "returns true for body is a JSON payload of the right format" do
+      body_for_json_availability_map = SampleData.json_availability_map_feed
+      expect(adapter.applicable?(body: body_for_json_availability_map, source: "")).to eq true
+      expect(adapter.applicable?(body: "", source: "")).to eq false
+    end
+  end
+
+  describe "#to_ical" do
+    it "returns iCal formated response" do
```
Comment (Azdaroth) [inline spec/vacation_rentals_ical_adapters/adapters/json_availability_map_adapter_spec.rb]: A small nitpick, I think it should be `formatted`.

**Comment by ZenCocoon** [comment]
Typo fixed. About the naming of this adapters neither are a standard and actually even the current Arkiane could be confusing as not including the API version number.

Regarding the `CSV Booked Days` is a new format that just happen to be very convenient and will be eligible only for file with the extension `booked_days` which feels rather selective and explicit.

Regarding the `JSON Availability Map`, availability map in this format is rather common, what can change is the actual JSON payload representation but again this is not app/channel specific per say so would not know how to name it more accurately here.

**Comment by Azdaroth** [comment]
Hmm, ok, anyway, the names of adapters are not the part of public API and they can be simply changed later, so let's keep it that way.

**Comment by ZenCocoon** [comment]
Thanks for the merge.

**Pull request merged**